src/components/AssetSearch/helpers/filterAssetsBySearchTerm/filterAssetsBySearchTerm.test.ts

```diff
@@ -1,38 +1,39 @@
+import { Asset } from '@shapeshiftoss/asset-service'
 import { filterAssetsBySearchTerm } from './filterAssetsBySearchTerm'
 
 const assets = [
   {
-    address: '0x9124248f2AD8c94fC4a403588BE7a77984B34bb8',
+    tokenId: '0x9124248f2AD8c94fC4a403588BE7a77984B34bb8',
     symbol: 'ETH',
     name: 'Ethereum'
   },
-  { address: '3Er42Ft2ZAyAbrpQ1PJUfgF7LbdJztcycd', symbol: 'BTC', name: 'Bitcoin' },
+  { tokenId: '3Er42Ft2ZAyAbrpQ1PJUfgF7LbdJztcycd', symbol: 'BTC', name: 'Bitcoin' },
```

Comment:
```suggestion
  { address: '3Er42Ft2ZAyAbrpQ1PJUfgF7LbdJztcycd', symbol: 'BTC', name: 'Bitcoin' },
```

Reply:
That is not correct. The data is set up to come back as `asset.tokenId` that is why that was changed and that is why these tests were braking.